api/gateway-operator/v2alpha1/controlplane_types.go

```diff
@@ -136,8 +136,36 @@ type ControlPlaneOptions struct {
 	//
 	// +optional
 	DataPlaneSync *ControlPlaneDataPlaneSync `json:"dataplaneSync,omitempty"`
+
+	// Translation defines the configuration for translating Kong configuration.
+	//
+	// +optional
+	Translation *ControlPlaneTranslationOptions `json:"translation,omitempty"`
+}
+
+// ControlPlaneTranslationOptions defines the configuration for translating
+// cluster resources into Kong configuration.
+type ControlPlaneTranslationOptions struct {
+	// CombinedServicesFromDifferentHTTPRoutes indicates whether the ControlPlane should
+	// combine services from different HTTPRoutes into a single service.
```

Comment:
```suggestion
	// combine services from different HTTPRoutes into a single Kong DataPlane service.
```

Maybe something like that to indicate that we don't think about Kong service here. WDYT?